pkg/cmd/clustertask/start.go

```diff
@@ -195,6 +196,7 @@ For passing the workspaces via flags:
 	c.Flags().StringVarP(&opt.PrefixName, "prefix-name", "", "", "specify a prefix for the TaskRun name (must be lowercase alphanumeric characters)")
 	c.Flags().StringVar(&opt.PodTemplate, "pod-template", "", "local or remote file containing a PodTemplate definition")
 	c.Flags().BoolVar(&opt.UseParamDefaults, "use-param-defaults", false, "use default parameter values without prompting for input")
+	c.Flags().BoolVarP(&opt.SkipOptionalWorkspace, "skip-optional-workspace", "", false, "skip to ask specification for optional workspace")
```

Comment:
how about adding the description -> `skips the prompt for optional workspaces` ?